src/components/SubNav/SubNav.tsx

```diff
@@ -0,0 +1,318 @@
+  "section.connect.primary-05",
+  "section.education.primary-05",
+  "section.locations.primary-05",
+  "section.research.primary-05",
+  "section.research-library.lpa-05",
+  "section.research-library.schomburg-05",
+  "section.research-library.schwarzman-05",
+  "section.whats-on.primary-05",
+  "dark.brand.primary-05",
+  "dark.section.blogs.primary-05",
+  "dark.section.books-and-more.primary-05",
+  "dark.section.connect.primary-05",
+  "dark.section.education.primary-05",
+  "dark.section.locations.primary-05",
+  "dark.section.research.secondary-05",
+  "dark.section.research-library.lpa-05",
+  "dark.section.research-library.schomburg-05",
+  "dark.section.research-library.schwarzman-05",
+  "dark.section.whats-on.primary-05",
+] as const;
+
+export type actionBackgroundColors = typeof actionBackgroundColorsArray[number];
+
+export const highlightColorsArray = [
+  "brand.primary",
+  "section.blogs.primary",
+  "section.books-and-more.primary",
+  "section.connect.primary",
+  "section.education.primary",
+  "section.locations.primary",
+  "section.research.primary",
+  "section.research-library-lpa.primary",
+  "section.research-library-schomburg.primary",
+  "section.research-library-schwarzman.primary",
+  "section.whats-on.primary",
+  "dark.brand.primary",
+  "dark.section.blogs.primary",
+  "dark.section.books-and-more.primary",
+  "dark.section.connect.primary",
+  "dark.section.education.primary",
+  "dark.section.locations.primary",
+  "dark.section.research.secondary",
+  "dark.section.research-library-lpa.primary",
+  "dark.section.research-library-schomburg.primary",
+  "dark.section.research-library-schwarzman.primary",
+  "dark.section.whats-on.primary",
+];
+export type highlightColors = typeof highlightColorsArray[number];
+
+export interface SubNavProps {
+  /** Additional class name for the `SubNav` component. */
+  className?: string;
+  /**
+   * Primary actions displayed on the left side of the SubNav.
+   * Use SubNavButton and SubNavLink components, which mirror
+   * the DS Button and Link.
+   */
+  primaryActions: React.ReactNode;
+  /**
+   * Secondary actions displayed on the right side of the SubNav.
+   * Use SubNavButton and SubNavLink components, which mirror
+   * the DS Button and Link.
+   */
+  secondaryActions?: React.ReactNode;
+  /**
+   * The background color to be applied to the hover and active states
+   * of the SubNavLink and SubNavButton components.
+   * This allows for customization of the action items.
+   */
+  actionBackgroundColor?: actionBackgroundColors;
+  /**
+   * Custom color for SubNavLink, SubNavButton, and icons.
+   */
+  highlightColor?: highlightColors;
+  /**
+   * Optional unique ID for accessibility, allowing other components
+   * to reference this element.
+   */
+  id?: string;
+  /** Optional string used to identify and highlight an item. The value should
+   * match the id associated with one of the items.
+   */
+  selectedItem?: string;
```

Comment:
I think we should keep this here, but we aren't currently using it at all in the SubNav component, only in the children. I think we need to have selectedItem passed to the parent and for the parent to somehow pass it to the child, rather than passing this prop to the child directly, because there should only be one selectedItem and right now, the consuming app could pass selectedItem prop to every child.

Reply:
Also, what do you think about making this prop selectedId rather than selectedItem?

Reply:
I just spoke with Deepika. We are going to remove the `selectedItem` prop altogether and instead add a new `isSelected` prop that can be passed to `SubNavButton` and `SubNavLink`, similar to `isOutlined`. This approach would allow more than one child to be marked as selected, so the onus would be on consuming apps to configure the child components accordingly and only mark one child as selected. This will be noted in the docs.